app/src/main/java/com/example/filemanager/AllFilesViewModel.kt

```diff
@@ -1,21 +1,32 @@
 package com.example.filemanager
 
 import android.os.Environment
-import android.util.Log
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
-import java.io.File
 
 class AllFilesViewModel() : ViewModel() {
 
     val files = MutableLiveData<List<ListModel>>()
+    private val listRootDirectoryFiles: ListModel = ListModel(Environment.getExternalStorageDirectory())
+    var currentFile: ListModel? = listRootDirectoryFiles
 
     fun fetchFiles() {
```

Comment:
in my previous comment I was refering to rename the `fetchFiles` method to `listRootDirectoryFiles`

Reply:
oh, okay, my bad